### PR TITLE
Improvements and restoration of frontend styles

### PR DESCRIPTION
**Pull request by G1t-Happens**
For practice purposes, I've worked on improving the frontend design and in some cases, restored the old design as well.
**Additional potential improvements:** Continue to externalize variables and enhance the use of SCSS variables.
If there is no interest, simply close the PR😃.
See pictures for reference:

A1
A2
A3
A4

### CODE REVIEW

**Comment by stritti** [comment]
Great to see, you continued my try to improve usage of SCSS.

File: frontend/src/assets/style/_variables.scss [hunk 1]
```diff
@@ -35,15 +39,21 @@
   --preparePageImportUserStoriesButton: #30a858;
   --preparePageActiveCardSet: black;
   --preparePageInActiveCardSet: #e27712;
-  --preparePageTimerBackground: #974f0d3d;
+  --preparePageTimerBackground: rgba(192, 175, 156, 0.4);
+  --preparePageNotSelectedTabBackground: rgba(192, 175, 156, 0.4);
 
   /*Session Page*/
-  --textAreaColour: #c3c7bd;
+  --textAreaColour: rgba(192, 175, 156, 0.4);
```
Comment (stritti) [inline frontend/src/assets/style/_variables.scss:46]: Should be replaced by basic SCSS variable which is used for input fields, I think.

File: frontend/src/assets/style/_variables.scss [hunk 2]
```diff
@@ -84,14 +97,22 @@
...
+  --textAreaColour: rgba(137, 152, 187, 0.4);
   --textAreaColourHovered: #a8e8f6b9;
+  --estimateButtonBorder: #ccd5da;
 
   /*other Stuff*/
   --topNavigationBarColor: rgba(34, 34, 34, 0.863);
   --popUpMenu: rgb(201, 201, 201);
+
+  /*Editor*/
+  --editorToolbar-Switch-bg: #7a777773;
```
Comment (stritti) [inline frontend/src/assets/style/_variables.scss:114]: Kebab-style is common on CSS variables

File: frontend/src/components/MarkdownEditor.vue [hunk 3]
```diff
@@ -103,26 +103,26 @@ export default Vue.extend({
 }
 
 .toastui-editor-defaultUI-toolbar {
-  background-color: #7a777773;
+  background-color: var(--editorToolbar-Switch-bg);
 }
 .toastui-editor-defaultUI-toolbar button {
-  border: none;
+  border: 12px;
```
Comment (stritti) [inline frontend/src/components/MarkdownEditor.vue:109]: Probably there is a variable at bootstrap, which could be used here too?

**Review by stritti — COMMENTED** [review]
I think many vars which currently exist on our list could be replaced by the already defined vars of bootstrap. This would make it more save, when we update something like border style in future.

**Review by SponsoredByPuma — APPROVED** [review]

**Comment by G1t-Happens** [comment]
> I think many vars which currently exist on our list could be replaced by the already defined vars of bootstrap. This would make it more save, when we update something like border style in future.

Hi @stritti, I also think that it would be best to control a large part of the styling centrally, using Bootstrap's SCSS variables. However, I believe that the process of migrating most of our custom stylings to Bootstrap SCSS variables will take some time and effort, which is why I plan to include it in a separate pull request in the future, in case we want to merge this one in the meantime.

**Review by stritti — APPROVED** [review]

**Pull request merged**